### DISCUSSION TITLE
New roger beep

### DISCUSSION THREAD

**Discussion by Axeminator** [egzumer/uv-k5-firmware-custom#402] [Ideas]
Please add other mdc1200 roger beep like that
 https://youtube.com/shorts/CSclPz9ICTs?feature=shared

**Comment by ErikS-web** [comment]
Why don't you buy that brand of radio.

There is only 60Kb for all this nice functionality

**Reply by Axeminator** [reply]
Maybe you can replace it with others.  If not, thank you for your answer.  I wish I could do it myself.

**Comment by Arikshten** [comment]
Is that not a data burst, rather than a roger tone?
